Specs/DateTime/English/TimeZoneExtractor.json

```diff
@@ -206,19 +206,19 @@
   {
     "Input": "It is the outcome of the vote that counts.",
     "Comment": "This case is to verify that the utc substring won't be extracted from the word outcome.",
-    "NotSupported": "javascript, python",
+    "NotSupported": "javascript, python, java",
```

Comment:
These were introduced for the C# after we started the migration. Since we kept porting from the same C# version (ignoring later commits) this weren't supported.
However, we can do a Regex/Logic update in this PR and enable the tests. This also happened with other extractors and we keep count of these new skipped tests.